syntax = "proto3";

package qtprotobufnamespace.tests;

message SimpleStringMessage {
    string testFieldString = 6;
}

